discord/ui/view.py

```diff
@@ -321,7 +322,8 @@ async def on_timeout(self) -> None:
                 message = self.message
 
             if message:
-                m = await message.edit(view=self)
+                with contextlib.suppress(discord.NotFound, discord.Forbidden):
+                    m = await message.edit(view=self)
                 if m:
```

Comment:
`m` could become unbound here, no ? Maybe a `try: except: else:` type pattern could fit better here.

Reply:
Works too, just because in general it's best to have the least amount of code possible in a try block (here contextlib.suppress), although it doesn't change anything in this particular case. Both should work fine